misk-admin/web/tabs/config/src/routes/index.tsx

```diff
@@ -6,6 +6,10 @@ const routes = (
   <div>
     <Switch>
       <Route path="/_admin/config/" component={TabContainer} />
+      <Route
```

Comment:
Thankfully this ends up being the only Misk-Web code change required of existing tabs to work in the new dashboard implementation.